instagram/src/components/LikesContainer/LikesContainer.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './LikesContainer.css';
+
+
+class LikesContainer extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            likes: this.props.likes,
+            clicked: false,
+        }
+    }
+
+    changeLikes = () => {
```

Comment:
Again you're keeping track of valuable information here far down the tree.  What if you wanted to use these likes somewhere else in the application (assuming you kept adding features to this project).  Then you'd have to send this data back up to App.js somehow only to pass it down to something else.